docs/usage.md

```diff
@@ -31,7 +31,13 @@
   * [`--save_align_intermeds`](#--save_align_intermeds)
   * [`--skip_alignment`](#--skip_alignment)
 * [Coverage tracks](#coverage-tracks)
+  * [`--skip_bigwig`](#--skip-bigwig)
```

Comment:
These were removed because they have been put in a table and so won't have explicit bullets in the relevant sections so the links won't work.